devnet/substrate_alice.Dockerfile

```diff
@@ -0,0 +1,55 @@
+# Copyright 2022 ChainSafe Systems (ON)
+# SPDX-License-Identifier: LGPL-3.0-only
+
+FROM golang:1.17 as openmetrics
+ARG METRICS_NAMESPACE=substrate.local.devnet
+
+WORKDIR /devnet
+
+COPY ./devnet/go.mod ./devnet/go.sum ./
+RUN go mod download
```

Comment:
do we need this? won't `go run ...` on line 13 just auto-download dependencies?

Reply:
This is for Docker layer caching, so dependencies are cached and rebuilds will be faster.